src/components/sidebar/types.ts

```diff
@@ -22,10 +24,7 @@ export interface MenuItem {
 
 export interface SidebarProps {
   /** Optional { text, url } to use for the "← Back to..." link at the top of the sidebar */
-  backToLink?: {
-    text: string
-    url: string
-  }
+  backToLinkProps?: SidebarBackToLinkProps
```

Comment:
Referencing newly created `SidebarBackToLinkProps` as the source of truth for this property.